lighthouse-core/report/styles/report.css

```diff
@@ -933,9 +933,9 @@ summary {
 
 .subitem-result__points {
   margin-top: calc((var(--subitem-line-height) - var(--subitem-font-size) - 4px) / 2);
-  background-color: #000;
+  background-color: fff;
   padding: 2px 4px;
-  color: #fff;
+  color: #000;
```

Comment:
IMO, this is pretty hard to read. Since the report is going through a bit of a redesign, let's keep it at `#fff` for now. That's more consistent with the rest of the report.